README.md

```diff
@@ -150,6 +159,33 @@ simp_bolt::config::modulepath: /path/to/modules
 To apply an existing manifest, `su` to the bolt user and execute
 `bolt apply <manifest> --nodes <NODE NAME(S)> --password --sudo-password`.
 
+### Within the SIMP Omni-Environment
```

Comment:
After re-reading this, and looking over the module, I think that this should be removed from this README and either added to `simp-doc` or (preferrably) as a task in the `simp-core` repository.